SystemMicrocontroller_Sensors/SystemMicrocontroller_Sensors Debug.launch

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Do this file needs to be included in project?

Reply:
Will be removed in next commit